client/src/components/page/Loading/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,101 +1,119 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`components/Loading renders the Loading component 1`] = `
-<Ripple
+<Loader
   borderColor="#ff5d00"
   borderStyle="solid"
   borderWidth="4px"
   height="64px"
   minHeight="80vh"
   width="64px"
 >
-  <div
-    className="lds-ripple-parent"
-    style={
-      Object {
-        "minHeight": "80vh",
-      }
-    }
+  <Ripple
+    borderColor="#ff5d00"
+    borderStyle="solid"
+    borderWidth="4px"
+    height="64px"
+    minHeight="80vh"
+    width="64px"
   >
     <div
-      className="lds-ripple"
+      className="lds-ripple-parent"
       style={
         Object {
-          "height": "64px",
-          "width": "64px",
+          "minHeight": "80vh",
         }
       }
     >
       <div
+        className="lds-ripple"
         style={
           Object {
-            "borderColor": "#ff5d00",
-            "borderStyle": "solid",
-            "borderWidth": "4px",
+            "height": "64px",
+            "width": "64px",
           }
         }
-      />
-      <div
-        style={
-          Object {
-            "borderColor": "#ff5d00",
-            "borderStyle": "solid",
-            "borderWidth": "4px",
+      >
+        <div
+          style={
+            Object {
+              "borderColor": "#ff5d00",
+              "borderStyle": "solid",
+              "borderWidth": "4px",
+            }
           }
-        }
-      />
+        />
+        <div
+          style={
+            Object {
+              "borderColor": "#ff5d00",
+              "borderStyle": "solid",
+              "borderWidth": "4px",
+            }
+          }
+        />
+      </div>
     </div>
-  </div>
-</Ripple>
+  </Ripple>
+</Loader>
 `;
 
 exports[`components/Loading renders the Loading component when props are given 1`] = `
```

Comment:
With regard to https://kentcdodds.com/blog/effective-snapshot-testing we can get rid of this snapshot since in this case we should only be interested in testing the props passed to `Ripple` which has been done below, the assumption being that the component `Ripple` should have been tested by the author and we should not worry about its implementation